spyder/plugins/ipythonconsole/widgets/namespacebrowser.py

```diff
@@ -205,7 +205,7 @@ def _handle_execute_reply(self, msg):
             self.ipyclient.t0 = time.monotonic()
 
         # Handle silent execution of kernel methods
-        if info and info.kind == 'silent_exec_method' and not self._hidden:
+        if info and info.kind == 'silent_exec_method':
```

Comment:
Does this need jupyter/qtconsole#505?

Reply:
jupyter/qtconsole#505 needs this but not the other way around. Anyway you should never have _hidden == True when info.kind == 'silent_exec_method'